src/internal/index.js

```diff
@@ -10,6 +10,10 @@ import LoggerTransport from "~/logger/logger-transport-internal";
 
 import { executeCommand, unsubscribeCommand, unsubscribeAllCommands } from "./commandHandler";
 
+/* eslint-disable */
+// $FlowFixMe
+coucou_arnaud = 1;
```

Comment:
@IAmMorrow i'm not exactly sure if it's testing a bad case a boot (like a module crashes). i think it would be better to do `coucou_arnaud()`